src/Microsoft.ML.ImageAnalytics/Microsoft.ML.ImageAnalytics.csproj

```diff
@@ -14,4 +14,10 @@
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Buffers">
```

Comment:
This needs to be a `<PackageReference>`. Also, don't forget to update the `.nupkgproj` as well

Reply:
Updated .nupkgproj.